xwiki-commons-core/xwiki-commons-velocity/src/test/java/org/xwiki/velocity/tools/JSONToolTest.java

```diff
@@ -169,6 +169,13 @@ void serializeBean()
         assertFalse(json.contains("\"transientProperty\":\"transient\""));
     }
 
+    @Test
+    void serializeSpecialCharacters()
```

Comment:
maybe be more explicit like:  `serializeWhenHTMLCharacters()`?

Reply:
I've now named it `serializeForwardSlash` which should be quite explicit.